api-gen/extraction/index.ts

```diff
@@ -3,17 +3,16 @@ import {writeFileSync} from 'fs';
 import {NgtscProgram, CompilerOptions, createCompilerHost} from '@angular/compiler-cli';
 
 function main() {
-  const [moduleName, srcs, outputFilenameExecRootRelativePath] = process.argv.slice(2);
+  const [moduleName, entryPoint, srcs, outputFilenameExecRootRelativePath] = process.argv.slice(2);
```

Comment:
Maybe in a follow-up?

```suggestion
  const [moduleName, entryPointExecRootRelativePath, srcs, outputFilenameExecRootRelativePath] = process.argv.slice(2);
```